training/xgboost/structured/base/scripts/train-cloud.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -v
+
+echo "Submitting an AI Platform job..."
+
+TIER="BASIC" # BASIC | BASIC_GPU | STANDARD_1 | PREMIUM_1
+
+export MODEL_NAME="sklearn_taxi"
+
+PACKAGE_PATH=./trainer # this can be a gcs location to a zipped and uploaded package
+export MODEL_DIR=gs://${BUCKET_NAME}/${MODEL_NAME}
+
+gsutil mb gs://${BUCKET_NAME}
```

Comment:
Will this call cause an issue if they use an existing bucket?

Reply:
Yes, you get a `ServiceException: 409 Bucket ....... already exists.`, but the script keeps running.